components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/SAMLAssertionClaimsCallback.java

```diff
@@ -19,14 +19,14 @@
 
 import com.nimbusds.jwt.JWTClaimsSet;
 import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
```

Comment:
Why is the reason for using net.minidev.json.JSONObject and removing org.json.JSONObject? Is this done intentionally?

Reply:
Yes. org.json.JSONObject will not get signed correctly in signJWT. Only the net.minidev.json.JSONObject will do that. Since now claim types will return as any type not always string, if a JSONObject is passed it should be in type net.minidev.json.JSONObject to sign properly.